test/e2e/network_segmentation.go

```diff
@@ -364,6 +364,84 @@ var _ = Describe("Network Segmentation", func() {
 				),
 			),
 		)
+
+		Context("pod2Egress on a user defined primary network", func() {
+			const (
+				externalContainerName = "ovn-k-egress-test-helper"
+			)
+			var externalIpv4 string
+			BeforeEach(func() {
+				externalIpv4, _ = createClusterExternalContainer(
+					externalContainerName,
+					"registry.k8s.io/e2e-test-images/agnhost:2.45",
+					runExternalContainerCmd(),
+					httpServerContainerCmd(port),
+				)
+
+				DeferCleanup(func() {
+					deleteClusterExternalContainer(externalContainerName)
+				})
+			})
+
+			DescribeTable(
+				"can be accessed to from the pods running in the Kubernetes cluster",
+				func(netConfigParams networkAttachmentConfigParams, clientPodConfig podConfiguration) {
+					netConfig := newNetworkAttachmentConfig(netConfigParams)
```

Comment:
skip if not an IPv4 cluster

Reply:
for now all net-seg lanes are dualstack.. we should know pretty well if we hit issues on v4 :D